### PR TITLE
Change opacity of the delete icon in the search field to be more visible

### PR DESCRIPTION
**Pull request by ghost**
Here's a fix for the "erase"/"delete" icon on the search field.

|Original|Updated padding|Updated opacity|
|--------|------------------|-----------------|
|image|image|image|

Let me know what you think about it!

### CODE REVIEW

File: app/javascript/styles/mastodon/components.scss [hunk 1]
```diff
@@ -8221,7 +8222,7 @@ noscript {
 
   .search__input {
     border: 1px solid lighten($ui-base-color, 8%);
-    padding: 10px;
+    padding: 10px 38px 10px 10px;
     padding-inline-end: 28px;
```
Comment (Gargron) [inline app/javascript/styles/mastodon/components.scss:8226]: This is the value you want to change.

**Review by Gargron — CHANGES_REQUESTED** [review]

**Review by ghost — COMMENTED** [review]
It is.

**Comment by ClearlyClaire** [comment]
For some reason, I do not have that padding issue on my end:
image

Furthermore, the new padding seems a bit much:
image

I'm in favor of the opacity change, though!

File: app/javascript/styles/mastodon/components.scss (hunk 1 above)
Comment (ClearlyClaire) [reply app/javascript/styles/mastodon/components.scss:8226]: There might be some confusion, as I haven't seen you take this comment into account (maybe you haven't pushed your current branch?)

The value that needs to be changed if you want to change the padding to make more space around the button is `padding-inline-end`. However, I see in your screenshot that it has been ignored. What browser are you using?

Comment (ClearlyClaire) [reply app/javascript/styles/mastodon/components.scss:8226]: Gargron did not suggest any code change here but just pointed out that the line to change would not be the `padding` line but the `padding-inline-end` one.

Which version of Firefox are you using? And which version of Mastodon have you seen the issue on? I'm starting to believe you're trying to fix an issue that was recently fixed in #26113!

**Review by ClearlyClaire — APPROVED** [review]

**Review by Gargron — APPROVED** [review]

**Pull request merged**